Insurgency/insserver

```diff
@@ -71,19 +114,19 @@ consolelogdate="${consolelogdir}/${servicename}-console-$(date '+%d-%m-%Y-%H-%M-
 # Do not edit
 
 fn_runfunction(){
+functiondir="${rootdir}/functions"
```

Comment:
Haven't had a chance to create a branch for just this, but take a look at the way I did it.
https://raw.githubusercontent.com/jaredballou/linuxgsm/master/Insurgency/insserver
Please let me know if you want me to include the version checking (or if you want to do it another way), and I'll try to get a version put up this weekend with just the change for the downloader.